interface/main/about_page.php

```diff
@@ -33,12 +33,20 @@
 ?>
  <html>
   <head>
-  <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+  <link rel=stylesheet href="<?php echo $css_header;?>" type="text/css">
+  <link rel=stylesheet href="../themes/login.css" type="text/css">
+  <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-ui-1-11-4/themes/base/jquery-ui.min.css" />
+  <link rel="shortcut icon" href="<?php echo $webroot; ?>/interface/pic/favicon.ico" />
```

Comment:
Remove favicon (redundant):
(I can do this minor change when bring the code in)